tests/test_make.py

```diff
@@ -45,6 +45,14 @@ def dataclass_param(request: _pytest.fixtures.SubRequest) -> DataclassModule:
     return module
 
 
+@pytest.fixture
+def typeddict() -> None:
+    try:
+        from typing import TypedDict
+    except ImportError:
+        raise pytest.skip("No TypedDict support")
```

Comment:
This form would allow a typo to result in disabling all the relevant testing.  I'd rather see an explicit version check.  Also, I'd like this to work with `typing_extensions.TypedDict` as well so this fixture should be parameterized against both for 3.8+ and otherwise just `typing_extensions.TypedDict`.  Maybe something like...

```suggestion
typed_dict_classes: List[Type] = typing_extensions.TypedDict
if sys.version_info >= (3, 8):
    typed_dict_classes.append(t.TypedDict)

@pytest.fixture(params=typed_dict_classes)
def typed_dict_class(request) -> Type:
    return request.param
```

And then the tests using this need to actually use the value.  This does kill the hinting here in the tests, but we've kind of already done that with the `module` anyways.